source/common/network/listener_filter_buffer_impl.h

```diff
@@ -0,0 +1,84 @@
+#pragma once
+
+#include <atomic>
```

Comment:
Is anything from `atomic` used in this file? Same for `functional`.

Reply:
no, maybe my ide add that, let me remove it.